--- wezterm-9999/Cargo.toml	2025-11-07 16:14:15.222881261 +0100
+++ wezterm-9999/Cargo.toml	2025-11-07 16:16:52.880493174 +0100
@@ -4,7 +4,6 @@
   "bidi/generate",
   "strip-ansi-escapes",
   "sync-color-schemes",
-  "deps/cairo",
   "wezterm",
   "wezterm-blob-leases",
   "wezterm-cell",
@@ -276,8 +275,3 @@
 zbus = "4.2"
 zstd = "0.11"
 zvariant = "4.0"
-
-[patch.crates-io]
-# We use our own vendored cairo, which has minimal deps and should just
-# build via cargo.
-cairo-sys-rs = {path="deps/cairo", version="0.18.0"}


--- wezterm-9999/wezterm-font/Cargo.toml	2025-11-07 16:27:24.737551921 +0100
+++ wezterm-9999/wezterm-font/Cargo.toml	2025-11-07 16:28:05.857747499 +0100
@@ -15,7 +15,7 @@
 
 [dependencies]
 anyhow.workspace = true
-cairo-rs.workspace = true
+cairo-rs = {version="0.18", default-features=false}
 config.workspace = true
 encoding_rs.workspace = true
 enum-display-derive.workspace = true

